### DISCUSSION TITLE
Save menu command

### DISCUSSION THREAD

**Discussion by splisp** [moneymanagerex/moneymanagerex#6780] [Q&A] [answered]
Why is there no "Save Database" command under the "File" menu? Only "Save Database as..." is present. I understand the db is saved automagically when exiting, but it'd be nice to also be able to save it when desired.

**Comment by n-stein** [comment, marked as answer]
The db is not only saved when exiting -- the transactions are recorded in the database in real-time, so it is automatically saved every time you make a change. Thus, a "Save Database" command is unnecessary.

**Reply by splisp** [reply]
Oh, I didn't realize it. Great, good to know, thanks!
